tests/conftest.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+from app import create_app
+from app.db import db
+from flask.signals import request_finished
+from dotenv import load_dotenv
+import os
+from app.models.planet import Planet
+
+load_dotenv()
+
+@pytest.fixture
+def app():
+    test_config = {
+        "TESTING": True,
+        "SQLALCHEMY_DATABASE_URI": os.environ.get('SQLALCHEMY_TEST_DATABASE_URI')
+    }
+    app = create_app(test_config)
+
+    @request_finished.connect_via(app)
+    def expire_session(sender, response, **extra):
+        db.session.remove()
+
+    with app.app_context():
+        db.create_all()
+        yield app
+
+    with app.app_context():
+        db.drop_all()
+
+@pytest.fixture
+def client(app):
+    return app.test_client()
+
+@pytest.fixture
+def get_every_single_planet(app):
```

Comment:
Our fixture function names should describe the data they are setting up to be used in tests, because we want them to be flexible enough to be used in more than one test. It seems like this fixture might be named for a specific test based on the `test_get_every_single_planet` test. I would recommend renaming this something like `save_five_planets`, and it can be reused in any test where it would be useful to have 5 planets saved before the test starts.